### PR TITLE
Added basic driver for BH1750 (light sensor), board GY-302

### PR DESCRIPTION
**Pull request by conejoninja**
Driver for supporting the i²c BH1750 light sensor, usually a blue board with the text BH1750 on one side and GY-302 on the other side

### CODE REVIEW

File: drivers/i2c/bh1750_driver.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+		return err
+	}
+
+	return
+}
+
+// Halt returns true if devices is halted successfully
+func (h *BH1750Driver) Halt() (err error) { return }
+
+func (h *BH1750Driver) RawSensorData() (level int, err error) {
```
Comment (trevrosen) [inline drivers/i2c/bh1750_driver.go]: Comment string?

File: drivers/i2c/bh1750_driver.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	}
+	if err != nil {
+		return
+	}
+	level = int(buf[0])<<8 | int(buf[1])
+
+	return
+}
+
+func (h *BH1750Driver) Lux() (lux int, err error) {
```
Comment (trevrosen) [inline drivers/i2c/bh1750_driver.go]: Comment string?

File: drivers/i2c/bh1750_driver.go (hunk 1 above)
Comment (deadprogram) [reply drivers/i2c/bh1750_driver.go]: Agreed, we need for Godocs generation.

File: drivers/i2c/bh1750_driver.go (hunk 2 above)
Comment (deadprogram) [reply drivers/i2c/bh1750_driver.go]: Same as other comment, needed for godocs.

**Comment by conejoninja** [comment]
Thanks for the feedback, added the comments and the sensor to the readme.md list.
It should be ok now.

**Comment by trevrosen** [comment]
Merging now that @deadprogram's feedback has been addressed.

**Pull request merged**